AppStoreExample/AppStoreExample/ViewModels/AppSyncViewModel.swift

```diff
@@ -9,42 +9,48 @@
 import Foundation
 
 final class AppSyncViewModel {
+    
+    typealias Observer = (State) -> Void
```

Comment:
Since this typealias is only used in single place I don't think it deserve a typealias